static/less/style.less

```diff
@@ -194,20 +194,66 @@ img {
         margin-left: 1.0rem;
         margin-right: 1.0rem;
       }
+      @media only screen and (min-device-width : 320px) and (max-device-width : 768px) {
+        float: none !important;
+      }
+    }
+    @media only screen and (min-device-width : 320px) and (max-device-width : 768px) {
+      visibility: hidden;
+      opacity: 0;
+      max-height: 0;
+      z-index: 5;
+      top: 5rem;
+      right: 0;
+      width: 100%;
+      position: absolute;
+      background-color: rgba(254,254,254 ,0.98);
+      padding: 0;
+      border-bottom: solid 2px #E2DFE1;
+      transition: opacity 0.25s, max-height 0.15s linear;
     }
   }
 }
 
-@media only screen and (min-device-width : 320px) and (max-device-width : 480px) {
-  a.navigation-title {
-    font-size: 0rem;
+#menu-control {
+  display: none;
+}
 
-    &::after {
-      content: '~';
-      font-size: 2.4rem;
-      text-align: center;
-      margin-left: -1.4rem;
-    }
+.btn-mobile {
+  display: none;
+}
+
+@media only screen and (min-device-width : 320px) and (max-device-width : 768px) {
+  .btn-mobile {
+    display: block;
+    font-size: 2.5rem;
+    color: black;
+    cursor: pointer;
+    padding-top: 1.5rem;
```

Comment:
I would suggest `margin-top` instead of `padding-top`. It will produce the same result, but if in the future someone wants to enhance the button in a way, let's say give it a border, background, etc then it won't be out of place.

Reply:
Or `line-height: 6rem;`